doc/source/WorkingWithScenes/LightLinking/index.md

```diff
@@ -0,0 +1,176 @@
+
+
+### Linking by set ###
+
+In setups with more than a few lights, managing a set expression with multiple light locations is prone to user error, and tedious to maintain. That's why it's usually better to link objects to one or more sets containing lights, as the term _set expression_ suggests. Put another way, you can perform **set**-centric light linking. The end result would still be the same, in that specific objects are linked to specific lights.
+
+A light can be added to any number of custom sets using the _Sets_ plug on the light's node, or a separate Set node. Custom sets are separate and do not affect the "defaultLights" set.
+
+![A Set node downstream of a light node in the Graph Editor](images/interfaceLightSetGraphEditor.png "A Set node downstream of a light node in the Graph Editor")
+
+![A Set node in the Node Editor](images/interfaceLightSetNodeEditor.png "A Set node in the Node Editor")
+
+To link one or more lights by a set, simply add it to the set expression.
+
+![A light linking set expression with a set](images/taskLightLinkingSetExpressionSet.png "The light linking set expression with a set")
+
+Be aware that if an object belongs to multiple sets with light linking, the object will only inherit the `linkedLights` attribute from the last set that was linked in the graph.
+
+![A graph where an object belongs to multiple sets, each with different linked lights](images/illustrationLightLinkingMultipleSets.png "A graph where an object belongs to multiple sets, each with different linked lights")
+
+
+## Example Scenario ##
+
+Let's explore an example scenario where you would need to link lights.
+
+Picture a scene where a giant robot attacks a highly detailed metropolitan city at night. For dramatic effect, let's say we want the robot's eyes to shine with some menacing red highlights. What's more, on the robot's chest is affixed the logo of its evil creator, and we want to make sure the audience can just make out the logo, regardless of the ambient illumination. With a photorealism mindset, both of these lights would be conceits, as they would have no plausible light source. Even so, with their inclusion, the scene becomes more expressive and evocative.
+
+![A preview of the example scenario in the Viewer](images/exampleGaffyAttacksScene.png "A preview of the example scenario in the Viewer")
+
+To achieve both of these highlights, we could add special lights in front of the robot's eyes and chest. But, by default, these lights would also illuminate the rest of the robot, and everything else in the scene, however imperceptibly. In a scene of this scale and complexity, the renderer would waste considerable resources calculating rays and their reflections between the special lights and the nearby buildings, streets, cars, crowds, debris, and other objects.
```

Comment:
Perhaps simplify this to something along the lines of "...everything else in the scene. This may not achieve our artistic intent and could also affect render times and memory usage." - instead of in the next paragraph?

Reply:
OK. Is it the tone you're concerned about?

Reply:
Just as the other comment really - it perhaps tends more to the 'why you might' than the 'how'...  In production I'd imagine a shot like this would probably make use of a whole bunch of complexity-management strategies.